### PR TITLE
Some features + some fixes

### PR DESCRIPTION
**Pull request by darkweaver87**
Hello,

Here are some non breaking modifications to make it work on an up to date mesos/marathon environment. I also added some features to make it compliant with our naming conventions :-)
Please accept those.

Rémi

### CODE REVIEW

**Comment by chiradeep** [comment]
I'm on vacation right now, will test and merge next week

**Comment by darkweaver87** [comment]
ok no problem :-)

File: main.py [hunk 1]
```diff
@@ -107,6 +109,8 @@ def cfg_file_driver(netskaler, cfg_file):
         cfg_file_driver.configure_ns_for_app(app)
 
 if __name__ == "__main__":
+    if re.match('^[a-zA-Z0-9+/=]+$', os.environ['APP_INFO']):
```
Comment (chiradeep) [inline main.py]: Can you explain this? I don't see any documentation in Marathon about this. When I tried it with  the latest marathon / mesos, it doesn't work. Removing the base64 encoding makes it work again.

**Comment by chiradeep** [comment]
Hi, this doesn't work (due to the base64 encoding change). I can't figure out why this (base64 enc) is required.

**Comment by darkweaver87** [comment]
Hello,

Sorry for late answer. This is because I didn't manage to start the application with marathon with a APP_INFO containing complexe json.
But I think I'll remove this and make APP_INFO be able  to be loaded from a json file.

Rémi

**Comment by wallnerryan** [comment]
cc @darkweaver87 @chiradeep is this project still valid? Looking to use this against mesos 1.9 and citrix 10.5

**Comment by chiradeep** [comment]
Tested against NetScaler 11, I believe but should be compatible. But please raise issues if you see them.

**Comment by wallnerryan** [comment]
@chiradeep thanks, We have MPX/VPX. Also is there a reason this PR was not merged?

**Comment by chiradeep** [comment]
Didn't work

**Comment by darkweaver87** [comment]
Hello wallnerryan,

Yes, we are using on our production systems since months. We have NS 10.5 and mesos 1.1.0.

Rémi

**Comment by wallnerryan** [comment]
@darkweaver87 Thanks. Just to clarify, you are running the `master` branch with " NS 10.5 and mesos 1.1.0"? 

We are running mesos 1.9, so i suspect some issues using this, if we find it useful and produce any fixes we will report/help contribute

**Comment by mikkodisini** [comment]
@adolforod Do you mind sharing your use case?  I am in the PM team with NetScaler.

**Comment by darkweaver87** [comment]
@wallnerryan Well yes, I'm running the master branch, this pull request included: https://github.com/Cdiscount/nitrox

**Comment by darkweaver87** [comment]
Cleaning some old PR, I don't think still need this.

**Pull request closed without merging**